resources/cluster-essentials/files/natsserviceroles.nats.crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
if this CRD is also valid for kyma 2.0 then it needs to be added to https://github.com/kyma-project/kyma/tree/main/installation/resources/crds to proper sub-folder.